common_functions.sh

```diff
@@ -39,6 +39,16 @@ all_arches="aarch64 armv7l ppc64le s390x x86_64 windows-amd windows-nano"
 # shellcheck disable=SC2034 # used externally
 all_packages="jdk jre"
 
+# All supported runtypes
+# shellcheck disable=SC2034 # used externally
+all_runtypes="build test"
+
+# Setting the OS's built in test
+PR_TEST_OSES="ubuntu alpine ubi"
+
+# setting default runtype to build
```

Comment:
Please expand the comment to define what are runtypes here before mentioning what is the default

Reply:
```
# `runtype` specifies the reason for running the script, i.e for building images or to test for a PR check
# Supported values for runtype : "build", "test"
# setting default runtype to build (can be changed via `set_runtype` function)
```

Added the comment above which specifies whats `runtype` is used for, supported values and how to set it